libexec/rtld-elf/rtld_c18n.c

```diff
@@ -832,8 +878,7 @@ _rtld_unw_getcontext_unsealed(uintptr_t ret, void **buf)
 struct jmp_args { uintptr_t ret1; uintptr_t ret2; };
```

Comment:
FYI this is specific to Morello's calling convention, so doesn't really belong in an MI file, but that's a pre-existing problem

Reply:
Yes. I'll sort this out while porting to RISC-V.